configs/chain-adapter.properties

```diff
@@ -0,0 +1,12 @@
+chain-adapter.rmqHost=d3-rmq
```

Comment:
is it possible to force chain-adapter use `rmq.properties` as well to decrease properties duplication?

Reply:
Duplication will go away if we put it inside `resources` folder.